src/components/SideNav/SideNav.tsx

```diff
@@ -1,18 +1,18 @@
 import * as React from "react";
 import { ReactElement, useState } from "react";
-import { NavigationProps } from ".";
+import { SideNavProps } from ".";
 import {
 	LinkWrapper,
-	SideNav,
+	SideNavStyle,
```

Comment:
Tip (you don't have to change this), the standard for styled components is "Styled_NameOfComponent_" in this case it would be something like StyledSideNav